frontend/src/routes/FacilityBooking/ViewFacility/index.tsx

```diff
@@ -273,7 +275,7 @@ export default function ViewFacility() {
                       params.facilityID,
                     ),
                   )
-                  history.push('/facility/booking/create')
+                  history.push('/facility/booking/create/' + params.facilityID)
```

Comment:
consider using backticks instead, with imported PATH (check for corresponding path)
```suggestion
                  history.push(`PATHS.../${params.facilityID}`)
```

Reply:
might be updated after wf merges, kiv